cmd/jujud/agent/model/manifolds.go

```diff
@@ -314,6 +315,36 @@ func Manifolds(config ManifoldsConfig) dependency.Manifolds {
 	return result
 }
 
+// CAASManifolds returns a set of interdependent dependency manifolds that will
+// run together to administer a CAAS model, as configured.
+func CAASManifolds(config ManifoldsConfig) dependency.Manifolds {
+	result := dependency.Manifolds{
+		// The first group are foundational; the agent and clock
+		// which wrap those supplied in config, and the api-caller
+		// through which everything else communicates with the
+		// controller.
+		agentName: agent.Manifold(config.Agent),
+		clockName: clockManifold(config.Clock),
```

Comment:
The clock manifold is generally regarded as a bad idea now. We should work to get rid of it.